rma/tests/test_rma.py

```diff
@@ -779,6 +779,8 @@ def test_04_supplier_rma(self):
         ).create({})
         wizard._create_picking()
         picking = self.rma_supplier_id.rma_line_ids._get_out_pickings()
+        partner = picking.partner_id
+        self.assertNotEqual(partner, False, "Partner is not defined or False")
```

Comment:
An small advice here, to do true/false assert, better user assertTrue/assertFalse.